pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -18,24 +18,39 @@ package tektonresult
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 
 	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
-	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
+	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
 	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	versionKey = "VERSION"
```

Comment:
what is this version environment key? From where do we get this?

Reply:
We get the version using this string. We don't need to redefine it. I see that we have done in other places but we can directly import `VersionEnvKey` from `v1alpha1`.

Reply:
Imported from `v1alpha1`.